Makefile

```
@@ -27,4 +27,4 @@ testsetup:
 	echo "running rosdep tests"
 
 test: testsetup
-	nosetests --with-coverage --cover-package=rosdep2 --with-xunit test
+	nosetests3 --with-coverage --cover-package=rosdep2 --with-xunit test
```

```suggestion
	nosetests --with-coverage --cover-package=rosdep2 --with-xunit test
```
It was my bad. I accidentally committed it.

---

If changes to the test runners are proposed I'd prefer them in a separate PR for ease of review.

---

Originally, the tests use `nosetests` (Python 2 version). I didn't intend to propose a test runner change. I changed it locally to try some stuff and committed it unintentionally. This suggestion reverts this change.